docs/data/material/migration/migrating-to-v6/migrating-to-v6.md

```diff
@@ -0,0 +1,292 @@
+
+<!-- #stable-snapshot -->
+
+- Node.js 18 (up from 12)
+- Chrome 109 (up from 90)
+- Edge 121 (up from 91)
+- Firefox 115 (up from 78)
+- Safari 15.4 in both macOS and iOS (up from 14 in macOS and 12.5 in iOS)
+- and more (see [.browserslistrc (`stable` entry)](https://github.com/mui/material-ui/blob/v6.0.0/.browserslistrc#L11))
+
+### Removed support for IE 11
+
+Support for IE 11 is completely removed, by dropping the legacy bundle and all IE 11 related code.
+This allows us to decrease bundle size and keep the scope manageable.
+If you need to support IE 11, you can use Material UI v5's [legacy bundle](https://v5.mui.com/material-ui/guides/minimizing-bundle-size/#legacy-bundle), but it won't get updates or bug fixes.
+
+## Update React & TypeScript version
+
+### Update React
+
+The minimum supported version of React is v17.0.0 (the same as v5).
+Use the snippet below to update your project (replace the `<version>` with the one you want):
+
+<codeblock storageKey="package-manager">
+
+```bash npm
+npm install react@<version> react-dom@<version>
+```
+
+```bash yarn
+yarn add react@<version> react-dom@<version>
+```
+
+```bash pnpm
+pnpm add react@<version> react-dom@<version>
+```
+
+</codeblock>
+
+### Update TypeScript
+
+The minimum supported version of TypeScript has been increased from v3.5 to 4.1.
+
+:::info
+We align with types released by [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) (published on npm under the `@types` namespace).
+
+We will not change the minimum supported version in a minor version of Material UI.
+However, we recommend not using a TypeScript version older than the lowest supported version by DefinitelyTyped.
+:::
+
+If your project includes these packages, you'll need to update them:
+
+- `@types/react`
+- `@types/react-dom`
+
+:::warning
+Make sure that your application is still running without errors, and commit the changes before continuing to the next step.
+:::
+
+## Breaking changes
+
+:::info
+This list is a work in progress.
+Expect updates as new breaking changes are introduced.
+:::
+
+### UMD bundle was removed
+
+To align with React 19's removal of UMD builds, Material UI has also removed its UMD bundle.
+This results in a reduction of the `@mui/material` package size by 2.5MB or 25% of the total package size.
+
+Instead, using ESM-based CDNs such as [esm.sh](https://esm.sh/) is recommended.
+For alternative installation methods, refer to the [CDN documentation](/material-ui/getting-started/installation/#cdn).
+
+### Chip
+
+Previously, the Chip component lost focus when the escape button was pressed, which differed from how other button-like components work.
+This issue has been resolved, and the chip component retains focus as expected.
+
+If you want to retain the previous behavior, add a custom `onKeyUp` handler, as shown below:
```

Comment:
```suggestion
If you want to keep the previous behavior, add a custom `onKeyUp` handler, as shown below:
```

Just because we used the verb "to retain" in the previous sentence already